### ISSUE TITLE
ExpandableText: Accessibility - "Show More" & popover content not read by screenreader, when used in Analytical tab inside a dialog

### ISSUE THREAD

**Issue by abhijeet181088** [SAP/ui5-webcomponents-react#5833]
### Describe the bug

We are using ExpandableText component in a column in AnalyticalTable, which is in a dialog. There are following accessibility issues.
1. "Show More" link is not read when focus goes to the link, instead screenreader reads "Close the popover" text
2. Popover's content is not read

### Isolated Example

https://stackblitz.com/edit/github-lg4jzd-cj6vlc

### Reproduction steps

1. Go to the stackblitz link provided
2. Click on "Open Dialog" button, dialog opens
3. Click on "Tab" to reach the first row's "Show More" button
4. Screenreader reads "Close the popover", which seems incorrect
5. Popover opens, but it's content is not read.

### Expected Behaviour

"Show More" / "Show Less" links should be read correctly when in focus.
Content of the popover should be read

### UI5 Web Components for React Version

1.28.1

### UI5 Web Components Version

1.24.0

### Browser

Chrome

### Operating System

macOS

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Hi @abhijeet181088 

[This PR](https://github.com/SAP/ui5-webcomponents-react/pull/5836) will fix the first issue. Regarding the second one we'll need more time to investigate. We'll let you know once there's an update.
